topics/icrc_34_get_delegation.md

```diff
@@ -1,50 +1,54 @@
-# ICRC-34: Get Global Delegation
+# ICRC-34: Delegation
 
 [![Status Badge](https://img.shields.io/badge/STATUS-DRAFT-ffcc00.svg)](https://github.com/orgs/dfinity/projects/31)
 [![Extension Badge](https://img.shields.io/badge/EXTENDS-ICRC--25-ffcc222.svg)](./icrc_25_signer_interaction_standard.md)
 [![Standard Issue](https://img.shields.io/badge/ISSUE-ICRC--34-blue?logo=github)](https://github.com/dfinity/wg-identity-authentication/issues/115)
 
 <!-- TOC -->
-* [ICRC-34: Get Global Delegation](#icrc-34-get-global-delegation)
+* [ICRC-34: Delegation](#icrc-34-delegation)
   * [Summary](#summary)
   * [Method](#method)
   * [Scope (according to the ICRC-25 standard)](#scope-according-to-the-icrc-25-standard)
-    * [Example RPC Request Permission](#example-rpc-request-permission)
-  * [Request](#request)
-    * [Example RPC Request](#example-rpc-request)
-    * [Response](#response)
-    * [Example RPC Response](#example-rpc-response)
+    * [Example RPC Permission Request](#example-rpc-permission-request)
+  * [icrc-25-supported-standards](#icrc-25-supported-standards)
+  * [Request params](#request-params)
+    * [Example RPC Delegation Request](#example-rpc-delegation-request)
+  * [Result](#result)
+    * [Example RPC Delegation Response](#example-rpc-delegation-response)
   * [Message Processing](#message-processing)
   * [Errors](#errors)
 <!-- TOC -->
 
 ## Summary
 
-When a relying party wants to authenticate as a user, it uses
-a [session key](https://internetcomputer.org/docs/current/references/ic-interface-spec/#signatures), and
-below `icrc34_get_global_delegation` method to obtain a delegation chain that allows the session key to sign for the
-user's global identity. The obtained delegation chain **MUST** be restricted by the signer to canister targets that
-trust the relying party following the [ICRC-28](./icrc_28_trusted_origins.md) standard.
+When a relying party wants to remove wallet approval prompts and sign on behalf of its users, it uses the icrc34_delegation 
+method described below. This method generates a delegation chain, an object with a stable identifier, allowing the relying 
+party to sign transactions on behalf of the user.
```

Comment:
I would rephrase this as `avoid going through the signer and user approval for each individual call`.
Also the fact that the relying party does not need to stay connected to the signer when it has a delegation is an interesting aspect that might be worth mentioning.

```suggestion
When a relying party wants to avoid going through the signer and user approval for each individual call, it can use the icrc34_delegation method described below. 

This method generates a delegation chain, an object with a stable identifier, allowing the relying party to sign transactions on behalf of the user without having to go through the signer and user approval. This also means the signer does not need to stay connected to the signer to make calls on behalf of the user.
```